src/compiler/crystal/macros.cr

```diff
@@ -153,6 +153,18 @@ module Crystal::Macros
     def stringify : StringLiteral
     end
 
+    # Returns a `SymbolLiteral` that contains this node's textual representation.
+    #
+    # ```
+    # macro test
+    #   {{ "foo".symbolize }}
+    # end
+    #
+    # puts test # prints "foo" (including the double quotes)
```

Comment:
Maybe is cleaner to show `{{ "foo".symbolize }} == :"\"foo\""  # => true`. 

I am not 100% why the double quotes are included in the symbol value though.

Reply:
I was wondering the same :) @asterite, would you please tell us why❓

Reply:
The textual representation of "foo" is "foo", and so a symbol with "foo" is `:"\"foo\"`. If you want `:foo` you need to do `"foo".id.symbolize`.

Reply:
Also `"foo".stringify` makes a string literal out of that, so it becomes `"\"foo\""`. The two methods are similar.